doc/manual/configuring.rst

```diff
@@ -0,0 +1,92 @@
+Networking and configuration
+============================
+
+Setting up core device networking
+---------------------------------
+
+For Kasli, insert a SFP/RJ45 transceiver (normally included with purchases from M-Labs and QUARTIQ) into the SFP0 port and connect it to an Ethernet port in your network. If the port is 10Mbps or 100Mbps and not 1000Mbps, make sure that the SFP/RJ45 transceiver supports the lower rate. Many SFP/RJ45 transceivers only support the 1000Mbps rate. If you do not have a SFP/RJ45 transceiver that supports 10Mbps and 100Mbps rates, you may instead use a gigabit Ethernet switch in the middle to perform rate conversion. 
+
+You can also insert other types of SFP transceivers into Kasli if you wish to use it directly in e.g. an optical fiber Ethernet network. Kasli-SoC already directly features RJ45 10/100/1000 Ethernet.
+
+If you purchased a Kasli or Kasli-SoC device from M-Labs, it will arrive with an IP address already set, normally the one requested in the web shop at time of purchase. Otherwise, if you chose not to make a request, the default IP M-Labs uses is ``192.168.1.75``. Once you can reach your core device, the IP can be changed at any time by running: ::
+
+  $ artiq_coremgmt -D [old IP] config write -s ip [new IP]
```

Comment:
``-D ip`` may be omitted if you have a ``device_db.py`` - that option just overrides that. After updating the address on the device, user should also update it in the ``device_db``.